main.cs

```diff
@@ -332,16 +333,27 @@ public static void LoadRoleClasses()
                 .GetTypes()
                 .Where(myType => myType.IsClass && !myType.IsAbstract && myType.IsSubclassOf(typeof(RoleBase)));
 
+            CustomRolesHelper.DuplicatedRoles = new Dictionary<CustomRoles, Type>
+            {
+                { CustomRoles.Vampiress, typeof(Vampire) },
+                { CustomRoles.Pestilence, typeof(PlagueBearer) },
+                { CustomRoles.Nuker, typeof(Bomber) },
+                { CustomRoles.NiceMini, typeof(Mini) },
+                { CustomRoles.EvilMini, typeof(Mini) }
+            };
+
+
             foreach (var role in CustomRolesHelper.AllRoles.Where(x => x < CustomRoles.NotAssigned))
             {
-                Type roleType = role switch // Switch role to FatherRole (Double Classes)
+                Type roleType;
+                if (CustomRolesHelper.DuplicatedRoles.ContainsKey(role))
```

Comment:
jsut check if (CustomRolesHelper.DuplicatedRoles.TryGetValue(role, out roleType)) and then assign the roletype with that variable

Reply:
nvm